src/lib/cloudinary.js

```diff
@@ -192,11 +192,11 @@ async function updateHtmlImagesToCloudinary(html, options = {}) {
     // Check to see if we have an existing asset already to pick from
     // Look at both the path and full URL
 
-    const asset = assets && Array.isArray(assets.images) && assets.images.find(({ publishPath, publishUrl } = {}) => {
-      return [publishPath, publishUrl].includes(imgSrc);
+    const asset =(imgUrl)=>assets && Array.isArray(assets.images) && assets.images.find(({ publishPath, publishUrl } = {}) => {
```

Comment:
since this is now a function, would you mind writing it out like this:

```
function name() {
```

i prefer it for readability

also i think it might make sense to update the function name to something that makes it clear what it's doing, even if it's someting like `getAsset` or `buildAsset`